docs/source/algorithms/Fit-v1.rst

```diff
@@ -611,6 +611,69 @@ Output:
     Constant 1: 2.86
     Constant 2: 14.14
 
+**Example - Fit to two data sets with one shared parameter, and different fit functions:**
+
+.. testcode:: shareFit3
+
+    import numpy as np
```

Comment:
```suggestion
    from mantid.simpleapi import *
    import numpy as np
```

When running this script in the mantid script editor you still need to import mantid.simpleapi.